sag_py_auth_brand/brand_jwt_auth.py

```diff
@@ -8,10 +8,8 @@
 from starlette.requests import Request
```

Comment:
this file is not entirely covered by unit tests anymore:
![image](https://github.com/SamhammerAG/sag_py_auth_brand/assets/41320207/9ee96a98-c70e-4c12-828b-60778778762f)

Reply:
fixed with: https://github.com/SamhammerAG/sag_py_auth_brand/pull/7/commits/42b37f62d34ec7f7b06334a215e9923ca8f282a7